src/js/components/Layer/stories/RTL.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Box, Grommet, Layer } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const RTLLayer = () => (
+  <Grommet theme={grommet} direction="rtl">
```

Comment:
```suggestion
  <Grommet theme={grommet} dir="rtl">
```

Reply:
I wonder how the story is still working although the prop isn't named as expected
![image](https://user-images.githubusercontent.com/6320236/73042159-65bbc680-3e1d-11ea-875e-93f4b58b62e5.png)

Reply:
The left is correct for "start" without `dir="rtl"`. I've fixed the story and now the Layer is on the right, correctly.